src/form/acr.jl

```diff
@@ -238,33 +290,39 @@ p[t_idx] ==  (g+g_to)*v[t_bus]^2 + (-g*tr-b*ti)/tm*(v[t_bus]*v[f_bus]*cos(t[t_bu
 q[t_idx] == -(b+b_to)*v[t_bus]^2 - (-b*tr+g*ti)/tm*(v[t_bus]*v[f_bus]*cos(t[f_bus]-t[t_bus])) + (-g*tr-b*ti)/tm*(v[t_bus]*v[f_bus]*sin(t[t_bus]-t[f_bus]))
 ```
 """
-function constraint_mc_ohms_yt_to(pm::_PMs.AbstractACRModel, n::Int, c::Int, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm)
-    constraint_mc_ohms_yt_from(pm, n, c, t_bus, f_bus, t_idx, f_idx, g, b, g_to, b_to, tr, ti, tm)
+function constraint_mc_ohms_yt_to(pm::_PMs.AbstractACRModel, n::Int, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm)
+    constraint_mc_ohms_yt_from(pm, n, t_bus, f_bus, t_idx, f_idx, g, b, g_to, b_to, tr, ti, tm)
 end
 
 
 ""
-function constraint_load_power_wye(pm::_PMs.AbstractACRModel, nw::Int, cnd::Int, load_id::Int, pd::Real, qd::Real)
-    _PMs.var(pm, nw, cnd, :pd)[load_id] = pd
-    _PMs.var(pm, nw, cnd, :qd)[load_id] = qd
+function constraint_load_power_wye(pm::_PMs.AbstractACRModel, nw::Int, load_id::Int, pd::Real, qd::Real)
+    for c in _PMs.conductor_ids(pm; nw=nw)
+        _PMs.var(pm, nw, :pd)[load_id][c] = pd[c]
+        _PMs.var(pm, nw, :qd)[load_id][c] = qd[c]
+    end
```

Comment:
I don't yet have a good sense if intermediate expressions should be in the solution dict or not.  At least thus far, I the solution dict only contains model variables and constraint duals, not intermediate expressions.

Still, becouse you are populating `var` here, you could also consider adding the `:pd` terms to the solution as well.

Thoughts?

Reply:
I'm not sure what to think of it either at this moment

Reply:
Agreed.  For now let's adopt the convention only true variables that occur in the JuMP model variables are put in the solution by default.